python/GafferArnoldTest/ArnoldShaderTest.py

```diff
@@ -363,6 +363,24 @@ def testAssignmentAttributeName( self ) :
 
 		self.assertEqual( a["out"].attributes( "/plane" ).keys(), [ "ai:surface"] )
 
+	def testLightFilterAssignmentAttributeName( self ) :
+
+		p = GafferScene.Plane()
+
+		s = GafferArnold.ArnoldShader( "light_blocker" )
+		s.loadShader( "light_blocker" )  # has isLightFilter tag in metadata
+
+		a = GafferScene.ShaderAssignment()
+		a["in"].setInput( p["out"] )
+		a["shader"].setInput( s["out"] )
+
+		self.assertEqual( s["attributeSuffix"].getValue(), "" )
```

Comment:
Question - should the attribute suffix be set to the name of the shader when we load it? That way I could assign a gobo and a blocker without them conflicting with each other, and would only need to use the suffix to allow two gobos to coexist. It would also mean that all assignment would match the pattern "ai:lightFilter:*" which I think is what we originally intended?